src/content-tags-drawer/ContentTagsDropDownSelector.jsx

```diff
@@ -69,71 +62,66 @@ const ContentTagsDropDownSelector = ({
     return (traversal[tag.value] && !traversal[tag.value].explicit) || false;
   };
 
-  useEffect(() => {
-    if (isTaxonomyTagsLoaded && taxonomyTagsData) {
-      setTags([...tags, ...taxonomyTagsData.results.filter(t => t.depth === level)]);
-      setNextPage(taxonomyTagsData.next);
-    }
-  }, [isTaxonomyTagsLoaded, taxonomyTagsData]);
-
   const loadMoreTags = useCallback(() => {
-    setCurrentPage(currentPage + 1);
-  }, [currentPage]);
+    setNumPages((x) => x + 1);
```

Comment:
When using a `set` function from a useState hook, you can pass in a function that gets the current value. This lets you simplify actions like incrementing. Now the `useCallback` actually has no dependencies :)

Reply:
oh sweet! TIL, didn't know that.